app/gui/src/project-view/components/ComponentBrowser/filtering.ts

```diff
@@ -280,13 +301,19 @@ export class Filtering {
   }
 
   /** TODO: Add docs */
-  filter(entry: SuggestionEntry, additionalSelfTypes: ProjectPath[]): MatchResult | null {
+  filter(entry: SuggestionEntry): MatchResult | null {
```

Comment:
On this occasion, add docs to this function.

Reply:
Added.